internal/api/mfa.go

```diff
@@ -345,6 +613,55 @@ func (a *API) challengePhoneFactor(w http.ResponseWriter, r *http.Request) error
 	})
 }
 
+func (a *API) challengeWebAuthnFactor(w http.ResponseWriter, r *http.Request) error {
+	ctx := r.Context()
+	db := a.db.WithContext(ctx)
+	user := getUser(ctx)
+	factor := getFactor(ctx)
+	params := &ChallengeFactorParams{}
+	if err := retrieveRequestParams(r, params); err != nil {
+		return err
+	}
```

Comment:
For avoidance of doubt,  in WebAuthn 

`VERIFY_DISABLED` should block the Challenge endpoint as wel;